tests/conftest.py

```diff
@@ -31,7 +35,7 @@ def python_main_environment(tmpdir_factory):
     return fn
 
 
-# minimal conda environment containing 
+# minimal conda environment containing
 # only pinned python with main and defaults channel
```

Comment:
These spaces at the end of the line were removed for pylint right?

Reply:
no, my vim auto removes white space at the end of the line when i save.  i hate trailing whitespace.